client/src/navbar/components/navbar-actions.tsx

```diff
@@ -25,15 +33,29 @@ export const NavbarActions = ({
     />
   ) : (
     <>
-      <Button
-        variant="outline"
-        size="sm"
-        className="gap-2"
-        onClick={() => setIsModalOpen(true)}
-      >
-        <LogInIcon size="16px" />
-        Log in
-      </Button>
+      <TooltipProvider>
+        <Tooltip>
+          <TooltipTrigger asChild>
+            <span>
+              <Button
+                variant="outline"
+                size="sm"
+                className="gap-2"
+                onClick={() => isOnline && setIsModalOpen(true)}
+                disabled={!isOnline}
+              >
+                <LogInIcon size="16px" />
+                Log in
+              </Button>
+            </span>
+          </TooltipTrigger>
+          {!isOnline && (
+            <TooltipContent>
+              <p>Connexion internet requise</p>
```

Comment:
l'app est en anglais partout donc chaud d'avoir qqchose comme
```suggestion
              <p>An internet connection is required to log in.</p>
```